cli/ops/test_dispatcher.rs

```diff
@@ -0,0 +1,40 @@
+use crate::test_dispatcher::TestMessage;
+use deno_core::error::AnyError;
+use deno_core::json_op_async;
+use deno_core::json_op_sync;
+use deno_core::serde_json;
+use deno_core::serde_json::json;
+use deno_core::serde_json::Value;
+use deno_core::BufVec;
+use deno_core::JsRuntime;
+use deno_core::OpState;
+use deno_core::ZeroCopyBuf;
+use serde::Deserialize;
+use std::cell::RefCell;
+use std::rc::Rc;
+use std::sync::mpsc::Sender;
+use std::sync::Arc;
+use std::sync::Mutex;
+
+pub fn init(rt: &mut deno_core::JsRuntime) {
+  super::reg_json_sync(rt, "op_send_test_message", op_send_test_message);
+}
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct SendTestMessageArgs {
+  message: TestMessage,
+}
+
+fn op_send_test_message(
+  state: &mut OpState,
+  args: Value,
+  _zero_copy: &mut [ZeroCopyBuf],
+) -> Result<Value, AnyError> {
+  let args: SendTestMessageArgs = serde_json::from_value(args)?;
+  let sender = state.borrow::<Sender<TestMessage>>().clone();
+
+  sender.send(args.message);
+
+  Ok(json!({}))
```

Comment:
Could we get away without an op if module execution returned a `v8` value?

Reply:
Yeah we could expose `tests` on Deno.internal and execute the tests from Rust instead (effectively moving the loop in runTests into Rust).

Would still use a channel like now but the sender would just be a local to `run_test`.

Reply:
Op has grown on me, I think it's fine.